plugins/dicom_viewer/webpack.helper.js

```diff
@@ -0,0 +1,18 @@
+module.exports = function(x) {
+    x.module.loaders.push({
+        test: /\.json$/,
+        loader: 'json',
+        include: ['node_modules'],
+    });
+    x.module.loaders.push({
+        test: /\.glsl$/,
+        loader: 'shader',
+        include: [/node_modules(\/|\\)vtk\.js(\/|\\)/],
+    });
+    x.module.loaders.push({
+        test: /\.js$/,
+        include: [/node_modules(\/|\\)vtk\.js(\/|\\)/],
+        loader: 'babel?presets[]=es2015,presets[]=react!string-replace?{"multiple":[{"search":"vtkDebugMacro","replace":"console.debug","flags":"g"},{"search":"vtkErrorMacro","replace":"console.error","flags":"g"},{"search":"vtkWarningMacro","replace":"console.warn","flags":"g"},{"search":"test.onlyIfWebGL","replace":"test","flags":"g"}]}',
```

Comment:
That's what I did at first, but vtk.js isn't present in node_modules when this is first accessed, so not really?